cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -165,10 +165,13 @@ func (t *GceTemplateBuilder) BuildNodeFromTemplate(mig Mig, template *gce.Instan
 		return nil, fmt.Errorf("could not obtain os-distribution from kube-env from template metadata")
 	}
 
-	ephemeralStorage, err := getEphemeralStorageFromInstanceTemplateProperties(template.Properties)
-	if err != nil {
-		klog.Errorf("could not fetch ephemeral storage from instance template. %s", err)
-		return nil, err
+	var ephemeralStorage int64 = -1
```

Comment:
I have used -1 as a default value where ephemeralStorage is not used.
I feel it is a better "bad" value, since 0 could still an eligible value to be used, but -1 storage is just not possible.
Hence, keeping it consistent, I would like to keep it -1.

Reply:
Fair enough.